kornia/augmentation/random_generator/random_generator.py

```diff
@@ -693,8 +693,8 @@ def center_crop_generator(
     src_h_half = src_h / 2
     src_w_half = src_w / 2
 
-    start_x = src_w_half - dst_w_half
-    start_y = src_h_half - dst_h_half
+    start_x = int(src_w_half - dst_w_half)
+    start_y = int(src_h_half - dst_h_half)
```

Comment:
@shijianjian will this affect to the subpixel precision when we crop with sampling ?

Reply:
Probably fine. But @dribnet, can I know the kornia and torch version you are using?

https://github.com/kornia/kornia/blob/d18c392a666b4d242413705848e7500cc7280046/kornia/augmentation/random_generator/random_generator.py#L713

The error line you referred is actually an empty line there.